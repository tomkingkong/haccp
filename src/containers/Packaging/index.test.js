import React from 'react';
import { shallow } from 'enzyme';

import { Packaging } from '.';

describe('Packaging', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Packaging ingredients={[]}/>);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});