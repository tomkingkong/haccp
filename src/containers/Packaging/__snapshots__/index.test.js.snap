// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Packaging should match snapshot 1`] = `
<div>
  <h2>
    Packaging
  </h2>
  <button
    onClick={[Function]}
  >
    Next Page
  </button>
</div>
`;
